src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -123,7 +127,9 @@ public InMoov(String n) {
   transient public OpenCV opencv;
   public Vision vision;
   transient public SpeechRecognizer ear;
+  public SpeechRecognition speechRecognition;
```

Comment:
SpeechRecognition class should not exist.  Please remove.   Why are you changing this ?  This was all worky before.